### ISSUE TITLE
incorrect "CONECT" between chains

### ISSUE THREAD

**Issue by hainm** [Amber-MD/pdb4amber#56]
From Diego in AMBER Mailing List 

```
I have experienced an issue with *pdb4amber* when dealing with SSBONDS
between chains, it renumbers the atoms after handling "TER", but it doesn't
update the CONECT record.

Thankfully, *Leap* produces a warning about this wrong connection.

The problem is restricted to AmberTools*17*, see bellow how to reproduce
for *1dwc.pdb*

# AmberTools 15
source /usr/local/amber14/amber.sh
pdb4amber -i 1dwc.pdb -o amber14.pdb
grep "ATOM   1196" amber14.pdb
ATOM   1196  SG  CYX H 148      55.729  28.382  19.687  1.00 10.07
 S
CONECT   24 1196

# AmberTools 17
source /usr/local/amber16/amber.sh
pdb4amber -i 1dwc.pdb -o amber16.pdb
grep "ATOM   1196" amber16.pdb
ATOM   1196  CB  CYX H 148      55.173  26.880  20.510  1.00 10.00
 C
CONECT   24 1196
```

**Comment by hainm** [comment]
uhm, work as expected.

```bash
pdb4amber -i 1dwc --pdbid -o at17.pdb
```

```python
In [1]: import parmed as pmd

In [2]: x = pmd.load_file('./at17.pdb')

In [3]: x.atoms[1195] # NOTE: the one reported in pdb CONECT use 1-based index.

Out[3]: <Atom SG [1195]; In CYX 147>

In [4]: x.atoms[23] # use 0-based index
Out[4]: <Atom SG [23]; In CYX 3>
```

**Comment by hainm** [comment]
still an issue.

**Comment by hainm** [comment]
fixed in #58
